hub/src/state.rs

```diff
@@ -77,7 +63,7 @@ impl<'a> HubMetadata {
                 metadata::ExecuteMsg::SetMetadata(new_metadata),
             )
             .map_err(ContractError::MetadataError)
-            .map(|_| Response::default())
+            .map(|_| Response::default()) // convert the glue result into a cosmwasm_std response
```

Comment:
Doesn't this line throw away the response value?

Reply:
yes it does but since there isn't any messages or events in the response value it can be discarded. Converting it by copying the messages and events from the response value into the new response value is unneeded since there isn't any data in the response

Reply:
Got it. In the future, we will want to pass the response type through, in case the module does start returning any data, submsgs or events. For now we can keep this.